src/QMCWaveFunctions/LCAO/LCAOrbitalSet.cpp

```diff
@@ -162,8 +162,8 @@ void LCAOrbitalSet::evaluateValue(const ParticleSet& P, int iat, ValueVector& ps
 }
 
 /** Find a better place for other user classes, Matrix should be padded as well */
-template<typename T, unsigned D>
-inline void Product_ABt(const VectorSoaContainer<T, D>& A, const Matrix<T>& B, VectorSoaContainer<T, D>& C)
+template<typename T, unsigned D, typename Alloc>
```

Comment:
Is this GEMM related change?

Reply:
yes, I can remove it for now along with the other changes to have dual-space coeffs (this function; added typedef to SpoSet.h; changed type of C and C_copy in LCAO).

Reply:
fixed